src/frontend/Felles/PersonHeader/PersonHeader.tsx

```diff
@@ -187,7 +188,10 @@ const PersonHeaderComponent: FC<{ data: IPersonopplysninger; behandling?: Behand
                         <EtikettFokus mini>Egen ansatt</EtikettFokus>
                     </ElementWrapper>
                 )}
-                {fullmakt.some((f) => erEtterDagensDato(f.gyldigTilOgMed)) && (
+                {fullmakt.some(
+                    (f) =>
```

Comment:
Vi burde trekke ut denne sjekken til en egen `const` med et beskrivende navn for hva sjekken validerer